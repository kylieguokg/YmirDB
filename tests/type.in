type no
set a
type a
set a 1 2 3
set b 0
append a b
get a
type a
pluck a 4
get a
type a
push a b
get a
type a 
pop a
get a
type a
bye
