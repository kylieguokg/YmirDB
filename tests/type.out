> no such key

> ok

> simple

> ok

> ok

> ok

> [1 2 3 b]

> general

> b

> [1 2 3]

> simple

> ok

> [b 1 2 3]

> general

> b

> [1 2 3]

> simple

> bye
